Content.Server/EntityEffects/Effects/PlantMetabolism/PlantAdjustPotency.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Content.Server.Botany.Systems;
+using Content.Shared.EntityEffects;
+
+namespace Content.Server.EntityEffects.Effects.PlantMetabolism;
+
+public sealed partial class PlantAdjustPotency : PlantAdjustAttribute
+{
+    /// <summary>
+    ///     Handles increase or decrease of plant potency.
+    /// </summary>
+
+    public override string GuidebookAttributeName { get; set; } = "plant-attribute-potency";
+
+    public override void Effect(EntityEffectBaseArgs args)
+    {
+        if (!CanMetabolize(args.TargetEntity, out var plantHolderComp, args.EntityManager))
+            return;
+
+        if (plantHolderComp.Seed == null)
+            return;
+
+        var plantHolder = args.EntityManager.System<PlantHolderSystem>();
+
+        plantHolderComp.Seed.Potency = Math.Max(plantHolderComp.Seed.Potency + Amount, 1);
```

Comment:
Potency is genetic, so you will need a `plantHolder.EnsureUniqueSeed(args.TargetEntity, plantHolderComp);` line above this as well.